### PR TITLE
bump builder image to golang-1.13 in dockerfiles

### PR DESCRIPTION
**Pull request by stlaz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
```
Comment (tnozicka) [inline Dockerfile]: I'd drop this whole file as we have done in other repos. There is no OKD yet and `registry.svc.ci.openshift.org/openshift/origin-v4.0:base` is actually OCP anyways

**Comment by marun** [comment]
@stlaz I'm assuming go.mod should be updated to 1.13 too?

https://github.com/openshift/service-ca-operator/blob/master/go.mod#L3

**Comment by stlaz** [comment]
@tnozicka apparently I can't simply remove the file since it's used in the build, but I should be able to do that once https://github.com/openshift/release/pull/6549 merges AFAIU

**Comment by tnozicka** [comment]
dockerfile FROM changes need to go in first, so you need to bump it there to 1.13 too and delete in a follow up. Or split the release PR into 2 (first to fix just the dockerfile location)

**Pull request merged**